integration_tests/test_stg_ga4__events.todo

```diff
@@ -37,5 +39,5 @@ def models(self):
     
     def test_mock_run_and_check(self, project):
         run_dbt(["build"])
-        #breakpoint()
+        breakpoint()
```

Comment:
Revert?

Reply:
I dropped this pytest for a few reasons I can get into. It was renamed to *.todo to keep the code handy but it won't run as part of the test suite. Just leaving it for future refactoring if I/we decide it adds value.